helm-chart/crds/retrytimeoutconfig.yaml

```diff
@@ -0,0 +1,98 @@
+    app.kubernetes.io/part-of: nginx-service-mesh
+spec:
+  group: nsm.nginx.com
+  scope: Namespaced
+  names:
+    kind: RetryTimeoutConfig
+    listKind: RetryTimeoutConfigList
+    shortNames:
+    - rtc
+    singular: retrytimeoutconfig
+    plural: retrytimeoutconfigs
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    schema:
+      openAPIV3Schema:
+        type: object
+        required:
+        - spec
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: Retry and timeout configuration.
+            type: object
+            required:
+            - targetRef
+            - default
+            properties:
+              targetRef:
+                type: object
+                description: The resource this retry and/or timeout applies to.
+                required:
+                - name
+                - kind
+                properties:
+                  kind:
+                    description: Kind of the targetRef.
+                    type: string
+                    enum:
+                    - Service
+                  name:
+                    description: Name of the targetRef.
+                    type: string
+                    minLength: 1
+                  namespace:
+                    description: Namespace of the targetRef.
+                    type: string
+                  port:
+                    description: 'Port of the targetRef to apply the retry and/or timeout to.
+                      If not specified, the policy will be applied to all ports.'
+                    type: integer
+                    minimum: 0
+                    maximum: 65535
+              default:
+                description: Default retry and timeout values.
+                type: object
+                properties:
+                  timeout:
+                    description: Time to wait for a request before aborting and returning an error.
+                    type: string
+                    pattern: "^[1-9][0-9]*(m|s|ms)$"
+                    default: 60s
+                  retry:
+                    description: Configuration for retrying a failed request.
```

Comment:
last thing. Can we say in this description that the default values are only considered if the retry block has been defined. Otherwise, no retries are performed.